engine/apps/api/permissions.py

```diff
@@ -117,29 +121,24 @@ def get_most_authorized_role(permissions: LegacyAccessControlCompatiblePermissio
     return min({p.fallback_role for p in permissions}, key=lambda r: r.value)
 
 
-def user_is_authorized(
-    user: "User",
-    required_permissions: LegacyAccessControlCompatiblePermissions,
-    required_basic_role_permission: LegacyAccessControlRole = None,
```

Comment:
this third argument here, `required_basic_role_permission` is _only_ used in `apps.api.permissions.BasicRolePermission.has_permission` ([here](https://github.com/grafana/oncall/pull/4784/files#diff-83058af8678d629be3d878a5947537c15c01abf6ef2eb7ba31d08b8febda0853R322)), updated that reference to simply use the new `user_has_minimum_required_basic_role` function instead